app/decorators/attachment_decorator.rb

```diff
@@ -2,11 +2,17 @@ class AttachmentDecorator < Draper::Decorator
   include Rails.application.routes.url_helpers
   include ActionView::Helpers::AssetTagHelper
   include ActionView::Helpers::UrlHelper
+  default_url_options[:host] = ::Rails.application.routes.default_url_options[:host]
```

Comment:
oh shoot I left this in accidentally. was trying to use some named routes in here and was looking at this https://github.com/drapergem/draper/wiki/Using-Rails-Path-Helpers-in-Draper-Decorators-with-Grape but ended up sticking with `object.url` instead of going for `proposal_attachment_url` since I looked in the controller and realized that just redirects to `object.url` anyway :)

not bad that it was left in but unnecessary so we can remove soon